appsec/third_party/CMakeLists.txt

```diff
@@ -59,13 +59,3 @@ endif()
 target_compile_definitions(zlibstatic PUBLIC ZLIB_CONST=1)
 target_include_directories(zlibstatic INTERFACE ${zlib_SOURCE_DIR} ${zlib_BINARY_DIR})
 
-include(ExternalProject)
-ExternalProject_Add(event_rules
-    GIT_REPOSITORY https://github.com/DataDog/appsec-event-rules.git
-    GIT_TAG 1.9.0
-    GIT_SHALLOW ON
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND ""
-    INSTALL_COMMAND "")
```

Comment:
Just out of curiosity: if you do not download the rules file from there anymore, do you have another process in place to keep that updated?

Reply:
It used to be like this. We had a local copy on the repository which was manually updated. When @Anilm3 moved appsec to this repo, he changed this. I had a with @cataphract yesterday and we both think it's more controller to have a local copy than downloading on the building process.